.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -191,6 +191,25 @@ const uploadPipeline = (pipelineContent: string | object) => {
       pipeline.push(getPipeline('.buildkite/pipelines/pull_request/check_next_docs.yml'));
     }
 
+    if (GITHUB_PR_LABELS.includes('ci:serverless-test-all')) {
```

Comment:
I thought we wanted to run all serverless tests all the time in soft failure mode, i.e. without specific labels. Is there a reason to go back to the labeling approach?

Reply:
I pushed https://github.com/elastic/kibana/pull/159310/commits/963fc78bdbd5f4e34196c2c747ef9a6432193844.  Probably a misunderstanding on my side